cmake/Findlibzmq.cmake

```diff
@@ -0,0 +1,97 @@
+#[========================================================================[.rst:
+Findlibzmq
```

Comment:
Any reason this isn't combined with the cppzmq library above?  Seems like with the other libraries the C and C++ versions were in the same Find<whatever>.cmake file.

Reply:
They're separate packages.

Reply:
👍